tests/test_e2e.py

```diff
@@ -163,7 +159,6 @@ def test_non_llm(self, protocol, setup_and_teardown):
         with pytest.raises(Exception):
             if protocol == "http":
                 pytest.xfail("Profiler does not support http protocol at this time")
-            self._profile(model, protocol=protocol)
```

Comment:
Need to remove this whole `with pytest.raises(Exception)` block: https://github.com/triton-inference-server/triton_cli/blob/07bd3379d709d66a1df97750a5dc40bd69f11c18/tests/test_e2e.py#L158-L161

Reply:
Ah, thanks. Removed!